tests/cypress/integration/IKImage.cy.js

```diff
@@ -1,33 +1,13 @@
-describe('ImageKit React SDK', () => {
+describe('IKImage Element', () => {
```

Comment:
Why have we modified existing test cases and changed the behavior?
Moreover, the modifications are incorrect. Unless the old test case and behavior were incorrect, this file should not be modified.